sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp

```diff
@@ -36,6 +37,68 @@ namespace sycl {
 inline namespace _V1 {
 namespace ext::oneapi::experimental {
 
+enum class group_algorithm_data_placement { blocked, striped };
+
+struct input_data_placement_key
+    : detail::compile_time_property_key<detail::PropKind::InputDataPlacement> {
+  template <group_algorithm_data_placement Placement>
+  using value_t =
+      property_value<input_data_placement_key,
+                     std::integral_constant<int, static_cast<int>(Placement)>>;
+};
+
+struct output_data_placement_key
+    : detail::compile_time_property_key<detail::PropKind::OutputDataPlacement> {
+  template <group_algorithm_data_placement Placement>
+  using value_t =
+      property_value<output_data_placement_key,
+                     std::integral_constant<int, static_cast<int>(Placement)>>;
+};
+
+template <group_algorithm_data_placement Placement>
+inline constexpr input_data_placement_key::value_t<Placement>
+    input_data_placement;
+
+template <group_algorithm_data_placement Placement>
+inline constexpr output_data_placement_key::value_t<Placement>
+    output_data_placement;
+
+inline constexpr input_data_placement_key::value_t<
+    group_algorithm_data_placement::blocked>
+    input_data_placement_blocked;
+inline constexpr input_data_placement_key::value_t<
+    group_algorithm_data_placement::striped>
+    input_data_placement_striped;
+
+inline constexpr output_data_placement_key::value_t<
+    group_algorithm_data_placement::blocked>
+    output_data_placement_blocked;
+inline constexpr output_data_placement_key::value_t<
+    group_algorithm_data_placement::striped>
+    output_data_placement_striped;
```

Comment:
Am I right that structures and variables are required for properties and we need to change the spec accordingly? Currently it says only about `input_data_placement` and `output_data_placement`

Reply:
Actually, these are not required, I added them just for convenience. Removed them for now, we can add them if needed with doc change.